src/core/trulens/core/utils/python.py

```diff
@@ -40,14 +41,14 @@
     Future = futures.Future
     """Alias for [concurrent.futures.Future][].
 
-    In python < 3.9, a subclass of [concurrent.futures.Future][] with
+    In python < 3.9, a sublcass of [concurrent.futures.Future][] with
```

Comment:
I am so surprised. How did you find these little things in such a large change.

Reply:
I just made another PR that contains more here but found most of these with some combination of `codespell` and a VS Code spell checking extension.
There are a few false positives so I didn't add it as a precommit hook.
https://github.com/truera/trulens/pull/1327